aspnetcore/security/key-vault-configuration/sample/Markup.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace KeyVaultConfigProviderSample
+{
+    internal static class Markup
+    {
+        internal const string Text = @"<!DOCTYPE html>

```

Comment:
Cool. I agree on the MVC being overkill. However, I hope we don't have to have the ever-so-tedious and painfully ugly `Console.WriteLine()` sample output. I just think this looks soooo much better. A case can be made for cross-teaching `app.Use()`. We'll have to see if the cats in charge will go for it.